schedule/sles4sap_offline_gnome_staging.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name:           sles4sap_offline_gnome
+description:    >
+  Graphical installation tests for SLES for SAP Applications without
+  SCC registration. Boot into Gnome.
+  Test SLES for SAP specific applications.
+vars:
+  APPTESTS: sles4sap/patterns,sles4sap/sapconf,sles4sap/saptune
+  DM_NEEDS_USERNAME: 1
+  EXCLUDE_MODULES: logs_from_installation_system
```

Comment:
This setting doesn't make sense with this approach. We can remove it as you don't have module in the schedule.

Reply:
fixed on ba01a8ac7